src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SystemProxyInfo.Unix.cs

```diff
@@ -6,11 +6,10 @@ namespace System.Net.Http
 {
     internal static class SystemProxyInfo
     {
-        // On Unix we get default proxy configuration from environment variables
+        // On Unix we get default proxy configuration from environment variables.
```

Comment:
Can we rename this file to .Linux.cs and say "Linux" here?

Reply:
I can makes those changes.  It still confuses me because there are TargetsUnix etc. in the CSPROJ file  and Configuration.props file.

Reply:
> It still confuses me because there are TargetsUnix etc. in the CSPROJ file and Configuration.props file.

TargetsUnix is true when compiling for either Linux or macOS.  TargetsLinux then further specializes for Linux, and TargetsOSX then further specializes for macOS.

Reply:
>TargetsUnix is true when compiling for either Linux or macOS. TargetsLinux then further specializes for Linux, and TargetsOSX then further specializes for macOS.

I understand that part I think.  But I'm confused then by why we build for "Unix" and not "Linux" if we're also specifying "OSX", i.e. 

https://github.com/dotnet/corefx/blob/0735c2d2b4c14f7ab6eb0e5b0b97a9f93abeb70f/src/System.Net.Http/src/Configurations.props#L1-L10

Reply:
That part doesn't make a lot of sense.  I expect it was a convenience / shortcut taken, before we specialized Linux vs macOS, and it just stayed in the build configurations.

Reply:
Ok. I'll leave it for now. I might change it someday later. Thanks for helping to explain it.

Reply:
FreeBSD also uses this so keep it as Unix (or some other name) makes more sense IMHO.

Reply:
So Unix = "Linux + OSX + FreeBSD" ?

Reply:
Does this mean that if we actually had netcoreapp-Linux instead of netcoreapp-Unix then FreeBSD wouldn't be built (unless we also had netcoreapp-FreeBSD)?

Reply:
> So Unix = "Linux + OSX + FreeBSD" ?

Yes, Unix == everything that's not Windows.

Reply:
So, System.Net.NetworkInformation has this:
https://github.com/dotnet/corefx/blob/0735c2d2b4c14f7ab6eb0e5b0b97a9f93abeb70f/src/System.Net.NetworkInformation/src/Configurations.props#L1-#L11

Reply:
I think based on this, I will revert my recent change just now and put it back to *.Unix since it really means "Unix" unless overridden by OSX.

Reply:
> Does this mean that if we actually had netcoreapp-Linux instead of netcoreapp-Unix then FreeBSD wouldn't be built (unless we also had netcoreapp-FreeBSD)?

It depends what the FreeBSD configuration actually tries to build.  I've not investigated as FreeBSD isn't supported in 3.0.  @wfurt would know.  There is a TargetsFreeBSD as well; TargetsUnix will be true when any of TargetsLinux, TargetsOSX, and TargetsFreeBSD are true, but TargetsLinux, TargetsOSX, and TargetsFreeBSD are all exclusive to each other.

Reply:
> since it really means "Unix" unless overridden by OSX.

Ok.  That's generally not what we mean with the .Unix.cs suffix, but that is how this condition is written, and I don't have a better name for the file if it's actually being used for all Unixes other than macOS.

Reply:
I will put the name back to *.Unix.cs.  But I was change the comment inside the file to be clearer.